easybuild/easyconfigs/l/libharu/libharu-2.3.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,46 @@
+# EasyBuild easyconfig
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>,
+#             Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the
+# policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# Modified for foss-2016b by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'CMakeMake'
+
+name = 'libharu'
+version = '2.3.0'
+
+homepage = 'http://libharu.org/'
+description = """libHaru is a free, cross platform, open source library for
+ generating PDF files."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = [' https://github.com/libharu/libharu/archive/']
```

Comment:
please drop the extra space before `https://`, there's absolutely no need for it?

Reply:
fixed in 35dd3e25cb